core/scripts/keystone/templates/oracle.toml

```diff
@@ -21,5 +21,7 @@ providerType = "ocr3-capability"
 telemetryType = "plugin"
 
 [onchainSigningStrategy]
-strategyName = 'single-chain'
+strategyName = 'multi-chain'
 [onchainSigningStrategy.config]
+aptos = "{{ aptos_key_bundle_id }}"
```

Comment:
This only works on my `aptos-init` branch since I fetch the extra keys, let's base the PR on that